### PR TITLE
Informatica generated commands - stable swagger version 2024-05-08

### PR DESCRIPTION
**Pull request by almasak**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [x] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [x] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/informatica/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Azure CLI Informatica Extension #
+This is an extension to Azure CLI to manage Informatica resources.
+
+## How to use ##
+Please add commands usage here.
```
Comment (kairu-ms) [inline src/informatica/README.md:5]: Please add content in this file.

File: src/informatica/azext_informatica/aaz/latest/informatica/data_management/organization/_create.py [hunk 2]
```diff
@@ -0,0 +1,588 @@
...
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "informatica data-management organization create",
+)
+class Create(AAZCommand):
+    """Create a InformaticaOrganizationResource
+    """
```
Comment (kairu-ms) [inline src/informatica/azext_informatica/aaz/latest/informatica/data_management/organization/_create.py:19]: Please add examples for commands in aaz-dev-tools before release, you can following this [doc](https://azure.github.io/aaz-dev-tools/pages/usage/workspace-editor/#command-examples)

File: src/informatica/azext_informatica/aaz/latest/informatica/data_management/organization/serverless_runtime/serverless/_resource_by_id.py [hunk 3]
```diff
@@ -0,0 +1,362 @@
...
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "informatica data-management organization serverless-runtime serverless resource-by-id",
```
Comment (kairu-ms) [inline src/informatica/azext_informatica/aaz/latest/informatica/data_management/organization/serverless_runtime/serverless/_resource_by_id.py:15]: What's the difference between this command and the `informatica data-management organization serverless-runtime show` command

File: src/informatica/azext_informatica/aaz/latest/informatica/data_management/organization/_create.py (hunk 2 above)
Comment (almasak) [reply src/informatica/azext_informatica/aaz/latest/informatica/data_management/organization/_create.py]: Added!

File: src/informatica/azext_informatica/aaz/latest/informatica/data_management/organization/serverless_runtime/serverless/_resource_by_id.py (hunk 3 above)
Comment (almasak) [reply src/informatica/azext_informatica/aaz/latest/informatica/data_management/organization/serverless_runtime/serverless/_resource_by_id.py:15]: Good catch! Currently, they are doing the same thing. I have removed _informatica data-management organization serverless-runtime show_ command for now.

File: src/informatica/azext_informatica/aaz/latest/informatica/data_management/organization/serverless_runtime/serverless/_resource_by_id.py [hunk 4]
```diff
@@ -0,0 +1,365 @@
...
```
Comment (kairu-ms) [inline src/informatica/azext_informatica/aaz/latest/informatica/data_management/organization/serverless_runtime/serverless/_resource_by_id.py:15]: Why remove the `informatica data-management organization serverless-runtime show` command? If they do the same thing, I recommend you keep the `informatica data-management organization serverless-runtime show` command. But if you prefer this command, I recommend rename it by the following one as it didn't use any id argument when using this command.
```suggestion
    "informatica data-management organization serverless-runtime serverless show-resource",
```

File: src/informatica/README.md (hunk 1 above)
Comment (almasak) [reply src/informatica/README.md]: I have added

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**